marketplacetests/marketplace_gaia_test.py

```diff
@@ -76,3 +76,18 @@ def install_certs(self):
     def wait_for_downloads_to_finish(self):
         self.marionette.switch_to_frame()
         self.wait_for_element_not_displayed(*self._statusbar_downloads_icon_locator)
+
+    def tap_install_button_of_first_paid_app(self, page):
+        """This method taps on the install button of the first paid app
```

Comment:
The summary should be a single line. See http://legacy.python.org/dev/peps/pep-0257/#multi-line-docstrings